postinstall.sh

```diff
@@ -52,43 +39,15 @@ cat >ios/edge-core/index.html <<HTML
   </head>
   <body>
     <script src="edge-core.js"></script>
-    <script>
-      var loading = 0
```

Comment:
Why was the code to do delayed loading of the plug-ins removed? Even though there are fewer plug-ins, it would still be helpful to delay loading the full bundle and let the GUI load with the core.

Reply:
Since there is only one now, the simple `async` prop on the script tag should now handle the delay. However, we should bench it to verify that this is the case.

Reply:
I went ahead and rolled it back for safety.